### PR TITLE
install: use pip via python

### PR DESCRIPTION
**Pull request by greut**
By using the grains it should be a tad more portable

### CODE REVIEW

File: docker/install.sls [hunk 1]
```diff
@@ -36,10 +38,11 @@ docker-package:
         {%- endif %}
         {%- if grains.os_family in ('Suse',) %}   ##workaround https://github.com/saltstack-formulas/docker-formula/issues/198
   cmd.run:
-    - name: /usr/bin/pip install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
+    - name: /usr/bin/pip{{ python:major }}.{{ python_minor }} install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
```
Comment (myii) [inline docker/install.sls]: Appears to be a typo: `{{ python:major }}` => `{{ python_major }}`.

Comment (greut) [reply docker/install.sls]:
```suggestion
    - name: /usr/bin/pip{{ python_major }}.{{ python_minor }} install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
```

File: docker/install.sls [hunk 2]
```diff
@@ -3,6 +3,8 @@
 {%- set docker_pkg_name = docker.pkg.old_name if docker.use_old_repo else docker.pkg.name %}
 {%- set docker_pkg_version = docker.version | default(docker.pkg.version) %}
 {%- set docker_packages = docker.kernel.pkgs + docker.pkgs %}
+{%- set python_major = grains['pythonversion'][0] | string %}
```
Comment (noelmcloughlin) [inline docker/install.sls]: Just checking one thing - I have never used that grain value - is it always populated? 
`{%- set python_major = 3 if not grains['pythonversion'][0] else ['pythonversion'][0]|string %}`
`{%- set python_minor = `7` if not grains grains['pythonversion'][1] else grains['pythonversion'][1] | string %` %}`
And being really paranoid is it always defined?

Comment (myii) [reply docker/install.sls]: Just a cross-reference to another PR dealing with the same issue right now:

* https://github.com/saltstack-formulas/libvirt-formula/pull/47/files#diff-5ec9631f5880e09dff9a91b7dde192c3

Comment (noelmcloughlin) [reply docker/install.sls]: @greut squash the commits anyway. I'm not saying anything wrong with implementation - we can assume all (most) OS has this grain and merge this if you have tested.

**Comment by greut** [comment]
It's much nicer to go with `python -m pip` rather than `pip` directly. I'm closing this, thanks for your time.

**Pull request closed without merging**